spec/support/publisher_helpers.rb

```diff
@@ -18,13 +18,35 @@ def fill_in_new_artefact_form(title:, slug:, format:)
     select format, from: "Format"
   end
 
+  def publish_artefact
+    confirm_action(link: "2nd pair of eyes", button: "Send to 2nd pair of eyes")
+    confirm_action(link: "Skip review", button: "Skip review")
+    confirm_action(link: "Publish", button: "Send to publish")
+  end
+
   def confirm_action(link:, button:)
     click_link link
     click_button button
 
     expect(page).to have_text('Transaction edition was successfully updated.')
   end
 
+  def add_part_to_artefact
```

Comment:
Looks like this relies on variables set outside the scope which would be inconsistent with other helper methods